tests/feature_tests/src/steps/report.py

```diff
@@ -131,13 +131,12 @@ def import_failed(context: Context) -> None:
     assert_equal("application/json", context.response.headers["Content-Type"])
 
 
-@then('the report {has_or_had} "{expected_number}" measurements')
-def get_measurements(context: Context, has_or_had: str, expected_number: str) -> None:
+@then('the report has "{expected_number}" measurements')
+def get_measurements(context: Context, expected_number: str) -> None:
     """Get the recent measurements of a report."""
-    if has_or_had == "had":
-        context.report_date = "2020-11-17T10:00:00Z"
-        context.min_report_date = "2020-11-16T00:00:00Z"
-
+    now = datetime.now(tz=UTC).replace(microsecond=0)
+    context.report_date = (now + timedelta(days=10)).isoformat()
```

Comment:
Because the endpoint now always needs a period (`min_report_date` and `report_date`). I tried to get rid of optional parameters that were basically only optional for sake of the tests.

Reply:
I was wondering about the +10 days in particular :-) But shouldn't be a problem